tests/data/token_indexers/token_feature_indexer.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=no-self-use,invalid-name
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.data import Token
+from allennlp.data.token_indexers import FeatureIndexer
+from allennlp.data.tokenizers.word_splitter import SpacyWordSplitter
+import numpy as np
+
+def lowercase_feature_extractor(token: Token):
+    if token.text[0].isupper():
+        return np.array([1.0], dtype=np.float32)
+    else:
+        return np.array([0.0], dtype=np.float32)
+
+class TestPosTagIndexer(AllenNlpTestCase):
+    def setUp(self):
+        super(TestPosTagIndexer, self).setUp()
+        self.tokenizer = SpacyWordSplitter(pos_tags=True)
```

Comment:
You're not using POS tags in this test, so I wouldn't load spacy using them.  Just the default spacy is fine.